docs/api/server/Environment-Variables.md

```diff
@@ -2,6 +2,461 @@
 
 # Environment Variables
 
-Documentation Forthcoming
+One App can be configured via Environment Variables:
 
-[☝️ Return To Top](#environment-variables)
\ No newline at end of file
+**Contents**
```

Comment:
should we also talk about how you can start setting this up where do you run them and when? do you have to restart once you this is setup? what happens if you need to change any of this?

Reply:
@PixnBits Any ideas on grouping ENV vars from an Ops perspective?

Reply:
Added a topic based ToC in b02bcb1